### PR TITLE
Gsoc 2018

### PR DESCRIPTION
**Pull request by petrkotas**
Changes proposed in this pull request:
This PR add new Google summer of code proposal for 2018.

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: @jelkosz @michalskrivanek

---
This change is [Reviewable](https://reviewable.io/reviews/ovirt/ovirt-site/1497)

### CODE REVIEW

**Comment by jelkosz** [comment]
@petrkotas hmmm, I dont think you wanted to add the "Add blog post about starting vm " commit into this PR :)

File: source/community/activities/summer-of-code.html.md [hunk 1]
```diff
@@ -42,6 +42,21 @@ When students approach you about the idea you listed:
 *   If you are redirecting students from your idea, please add [No longer taking applicants] to its title in the list below.
 *   Don't hesitate to reach out to the oVirt GSoC admins if you need help redirecting students.
 
+## oVirt Ideas for Google Summer of Code 2018
+
+### **Idea** Kubevirt/qemu-guest-agent
+
+**Description** The qemu-guest-agent is a useful tool to send system call to the
+guest operating system and KubeVirt project will benefit from it the same as
```
Comment (jelkosz) [inline source/community/activities/summer-of-code.html.md]: > to send system call to the guest operating system

"to communicate directly with the guest operating system" sounds better to me

Comment (petrkotas) [reply source/community/activities/summer-of-code.html.md]: Good catch, thanks.

**Comment by petrkotas** [comment]
If this is OK with you @mykaul, can we merge it, please?

File: source/community/activities/summer-of-code.html.md [hunk 2]
```diff
@@ -42,6 +42,24 @@ When students approach you about the idea you listed:
 *   If you are redirecting students from your idea, please add [No longer taking applicants] to its title in the list below.
 *   Don't hesitate to reach out to the oVirt GSoC admins if you need help redirecting students.
 
+## oVirt Ideas for Google Summer of Code 2018
+
+### **Idea** KubeVirt/qemu-guest-agent
```
Comment (mykaul) [inline source/community/activities/summer-of-code.html.md]: That should be under KubeVirt, not oVirt?

Comment (jelkosz) [reply source/community/activities/summer-of-code.html.md]: is kubevirt applying independently? Or you just meant the title?

Comment (mykaul) [reply source/community/activities/summer-of-code.html.md]: I don't know it does - it should - please find out promptly. If not, we'll host it here for a while...

Comment (jelkosz) [reply source/community/activities/summer-of-code.html.md]: I was under the impression it does not. @fabiand @rmohr is KubeVirt applying for google summer of code this year?

**Comment by sandrobonazzola** [comment]
Please resolve conflicts

**Comment by gregsheremeta** [comment]
You may also want to rebase on #1495, assuming that beats this one

**Comment by petrkotas** [comment]
@gregsheremeta Hi, I did the rebase. Thanks 👍

**Pull request merged**